spatialdata/_constants/_key.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, Callable
+
+
+class cprop:
+    def __init__(self, f: Callable[..., str]):
+        self.f = f
+
+    def __get__(self, obj: Any, owner: Any) -> str:
+        return self.f(owner)
+
+
+# TODO: use this class for various properties
+class Key:
+    """Class which manages keys in :class:`anndata.AnnData`."""
```

Comment:
What is this for?

Reply:
this is something that turned out to be quite handy for a bunch of use cases (specifically accessing anndata slots). For now is unused but I wanted to put it there for the first occasion to use it.

Reply:
Got it. Yes, please remove it before merging. I think we want to start out as lean as possible to reduce the clean up work we need to do in the future.